main.js

```diff
@@ -0,0 +1,214 @@
+const submitBtn = document.querySelector(".search");
+const current = document.querySelector(".current");
+const fiveDay = document.querySelector(".five-day");
+const searchInput = document.querySelector(".city-name");
+const clear = '<img src="http://openweathermap.org/img/wn/01d@2x.png">';
+const clouds = '<img src="http://openweathermap.org/img/wn/04d@2x.png">';
+const drizzle = '<img src="http://openweathermap.org/img/wn/09d@2x.png">';
+const thunderStorm = '<img src="">';
+const rain = '<img src="http://openweathermap.org/img/wn/10d@2x.png">';
+const snow = '<img src"http://openweathermap.org/img/wn/11d@2x.png">';
+const mist = '<img src="http://openweathermap.org/img/wn/50d@2x.png">';
+let currentResults = [];
+let fiveDayResults = [];
+let conditionImg;
+
+submitBtn.addEventListener("click", () => {
+  document.querySelector(".current").replaceChildren();
+  document.querySelector(".five-day").replaceChildren();
+  currentResults = [];
+  fiveDayResults = [];
+  conditionImg = "";
+  let city = searchInput.value;
+
+  if (!city) {
+    alert("Please enter valid city name.");
+  } else {
+    fetchWeather(city);
+  }
+
+  searchInput.value = "";
+});
+
+async function fetchWeather(city) {
+  try {
+    const currentWeather = await fetch(
+      `https://api.openweathermap.org/data/2.5/weather?q=${city}&appid=8ecfabbc5fb6be8c3ec37ba3114514e4&units=imperial`
+    );
+    const currentWeatherData = await currentWeather.json();
+    getCurrentWeather(currentWeatherData);
+
+    const fiveDayForecast = await fetch(
+      `https://api.openweathermap.org/data/2.5/forecast?q=${city}&appid=8ecfabbc5fb6be8c3ec37ba3114514e4&units=imperial`
+    );
+    const fDFData = await fiveDayForecast.json();
+    getFiveDayForcast(fDFData);
+  } catch (error) {
+    console.error("Error fetching data");
+    alert("Weather inquiry failed.");
+  }
+}
+
+function getCurrentWeather(currentWeatherData) {
+  currentResults.push({
+    city: currentWeatherData.name,
+    temp: Math.floor(currentWeatherData.main.temp),
+    conditions: currentWeatherData.weather[0].main,
+  });
+  renderCurrentWeather(currentResults);
+}
+
+function getFiveDayForcast(fDFData) {
+  const fiveDayHourly = fDFData.list;
+  for (let i = 0; i < fiveDayHourly.length; i = i + 8) {
+    fiveDayResults.push(fiveDayHourly[i]);
+  }
+  renderFiveDayForecast(fiveDayResults);
+}
+
+function renderCurrentWeather(currentResults) {
+  const temp = currentResults[0].city;
+  const city = currentResults[0].temp;
+  let conditions = currentResults[0].conditions;
+
+  for (let i = 0; i < currentResults.length; i++) {
+    if (conditions === "Clouds") {
```

Comment:
this could be optimized with a dictionary/object
```
const weatherIcons = {
  Clouds: clouds,
  Rain: rain,
 ... etc
};
conditionImg = weatherIcons[conditions] || mist;
```